### PR TITLE
One easy commit for the perfect pie... chart

### PR DESCRIPTION
**Pull request by jonathan-best**
Add internal vs external traffic pie charts to live and archive page

screen shot 2016-03-04 at 10 31 15

https://trello.com/c/t8kHkuoP

### CODE REVIEW

File: src/shared/handlers/ArticleRealtimeView.js [hunk 1]
```diff
@@ -522,25 +519,22 @@ class ArticleRealtimeView extends React.Component {
...
             >
-              <h5>FT Traffic Sources</h5>
+              <h5>Traffic Source</h5>
               <BarChart
-                data={refData}
-                keys={refKeys}
-                category={refID}
-                yLabel="Internal referrals"
+                data={extRefData}
+                keys={extRefKeys}
```
Comment (StuartFT) [inline src/shared/handlers/ArticleRealtimeView.js]: Data should be sorted based on volume?

Without (
https://ft-editorial-lantern-pr-515.herokuapp.com/articles/743d91b8-df8d-11e5-b67f-a61732c1d025/48 ) shows internal which is an important source kinda hidden at the bottom of the chart

Comment (jonathan-best) [inline src/shared/handlers/ArticleRealtimeView.js]: Thats a good idea this can be done as a separate feature/PR as this PR was just for the pie charts 😊

Comment (jonathan-best) [inline src/shared/handlers/ArticleRealtimeView.js]: @StuartFT Ive added this to a separate PR https://github.com/Financial-Times/lantern/pull/517 :+1:

**Comment by StuartFT** [comment]
:+1:

**Pull request merged**